opensrp-reveal/src/test/java/org/smartregister/reveal/fragment/BaseOfflineMapsFragmentTest.java

```diff
@@ -0,0 +1,131 @@
+package org.smartregister.reveal.fragment;
+
+import android.content.Context;
+import android.support.v7.app.AppCompatActivity;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.powermock.reflect.Whitebox;
+import org.robolectric.Robolectric;
+import org.robolectric.RuntimeEnvironment;
+import org.robolectric.shadows.ShadowToast;
+import org.smartregister.reveal.BaseUnitTest;
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.util.TestingUtils;
+
+import static io.ona.kujaku.services.MapboxOfflineDownloaderService.SERVICE_ACTION.DELETE_MAP;
+import static io.ona.kujaku.services.MapboxOfflineDownloaderService.SERVICE_ACTION.DOWNLOAD_MAP;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+/**
+ * Created by Richard Kareko on 1/31/20.
+ */
+
+public class BaseOfflineMapsFragmentTest extends BaseUnitTest {
+
+    @Rule
+    public MockitoRule rule = MockitoJUnit.rule();
+
+    @Captor
+    private ArgumentCaptor<String> stringArgumentCaptor;
+
+    private BaseOfflineMapsFragment.MapDownloadReceiver mapDownloadReceiver;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'mapDownloadReceiver' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)